### DISCUSSION TITLE
Reverse proxy for HTTPS web ui

### DISCUSSION THREAD

**Discussion by coc0nut** [haugene/docker-transmission-openvpn#2739] [Ideas]
It would be nice to be able to set an environment variable for base URI for transmission so that it easily can be put behind a reverse proxy... Is there a HOWTO for this? If so I would like a link for it :) 

Nice project!

**Comment by pkishino** [comment]
I have mine behind my Nginx reverse proxy via https but there’s nothing special to config, just forward to the 9091(or whatever you use) port and done

**Comment by edgd1er** [comment]
I have mine behind traefik , a bit tricky to set up, but once running , you can forget it. 
The easier part, proxy rules are set via labels on the transmission container. 
You can also re use all middleware that are already defined in traefik ( fail2ban, hsts, authentification, ip filtering, redirection, compression , ....)
